modules/EvseManager/Charger.hpp

```diff
@@ -201,7 +214,9 @@ class Charger {
     Everest::Thread mainThreadHandle;
 
     const std::unique_ptr<board_support_ACIntf>& r_bsp;
+    const std::vector<std::unique_ptr<powermeterIntf>>& r_powermeter_billing;
     const std::string& connector_type;
+    const std::string evse_id;
```

Comment:
could probably be a const&

Reply:
It is not obvious to me what guarantees `evse_id` outliving the Charger object in all circumstances. Hence I think it is better to have a non-reference and guarantee ownership here like it is currently done.